models/DataObject/Data/ElementMetadata.php

```diff
@@ -196,4 +196,11 @@ public function __toString(): string
     {
         return $this->getElement()->__toString();
     }
+
+    public function __wakeup(): void
```

Comment:
```suggestion
    public function __unserialize(): void
```
and avoid property type change, otherwise it would lead to bc-break

Reply:
something like
```
        foreach (get_object_vars($this) as $property => $value) {
            if ($property === 'elementId')){
                  $this->$property = (int) ( $data["\0*\0".$property] ?? $value);
                  continue;
            }
            $this->$property = $data["\0*\0".$property] ?? $value;
        }
```